### PR TITLE
Portofolio Desktop version

### PR DESCRIPTION
**Pull request by ramiaquista**
- Added Responsiveness for my personal portfolio

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -468,3 +474,269 @@ form > * {
...
+    padding-bottom: 100px;
+  }
+
+  .about-myself {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    margin-top: 10%;
+    padding-left: 6%;
+  }
```
Comment (safafa) [inline style.css]: - [ ] As you can see in the original design the About myself left section is smaller than the right section. it takes about 40% of the wrapper and the right takes like 60%
image
Yours looks the same size. Kindly fix it
image

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @ramiaquista , You did a great job so far 👏🏼 👏🏼 

- Well done on following the original design
- the website is responsive

However, here are some points that you need to work on to get your project approved.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css (hunk 1 above)
Comment (ramiaquista) [reply style.css]: Oh my bad, I am going to correct that! Thank's for the advice.

**Review by barackm — APPROVED** [review]
Hi @ramiaquista, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**